src/usePopper.js

```diff
@@ -41,7 +41,7 @@ export const usePopper = (
   const [state, setState] = React.useState<State>({
     styles: {
       popper: {
-        position: optionsWithDefaults.strategy || 'absolute',
```

Comment:
`optionsWithDefaults` already defaults `strategy` to `'absolute'`, so we don't need to do another check for a falsy value here.